.github/workflows/govulncheck.yaml

```diff
@@ -0,0 +1,37 @@
+on:
+  workflow_call:
+
+name: Govulncheck check
+
+permissions:
+  contents: read
+
+jobs:
+  govulncheck:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Install golang
+        run: |
+          ./tests/install_go.sh -f -p
+          echo "/usr/local/go/bin" >> "${GITHUB_PATH}"
+
+      - name: Install govulncheck
+        run: |
+          go install golang.org/x/vuln/cmd/govulncheck@latest
+
+      - name: Build all runtime binaries
+        run: |
+          cd src/runtime
+          make all
+        env:
+          SKIP_GO_VERSION_CHECK: "1"
+
+      - name: Run govulncheck on all runtime binaries
+        run: |
+          bash govulncheck-runner.sh
+        working-directory: ./.github/govulncheck/
```

Comment:
We don't typically use `working-directory` - let's pass the full script path to Bash.